src/core/checkedint.d

```diff
@@ -35,7 +35,7 @@ version (LDC)
     // llvm.(u)mul.with.overflow.i64 might fall back to a software implementation
     // in the form of __mulodi4, which only exists in compiler-rt and not
     // libgcc. Thus, we need to be sure not to emit it for now (see GitHub #818).
-    version (X86_64)
+    version (D_LP64)
```

Comment:
Seems reasonable for all 64-bit targets, right?

Reply:
The comment above states otherwise.

Reply:
ldc-developers/ldc#818 was about 32-bit x86 as expected. As there are no unittest linker issues with AArch64, at least that one doesn't seem to require avoiding the intrinsic.